Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -392,10 +392,13 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
             return controller
         }()
 
+        messageContentView.textView?.textContainer.maximumNumberOfLines = 6
```

Comment:
Rather than limiting it like this, it would be better to use the autolayout to truncate the message on its max allowed with.